projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss

```diff
@@ -61,30 +60,21 @@
       border: $border-1;
       border-radius: $border-10;
       box-shadow: none;
-    }
-
-    .sorting{
-      // nb-icon {
-      //   font-size: $fs-40;
-      //   font-weight: $fw-500;
-      // }
```

Comment:
We need this class for highlighting the button selected, let's not remove it from the css.

Reply:
This was duplicate code